pep-0597.rst

```diff
@@ -33,135 +34,140 @@ Using the default encoding is a common mistake
 Developers using macOS or Linux may forget that the default encoding
 is not always UTF-8.
 
-For example, ``long_description = open("README.md").read()`` in
-``setup.py`` is a common mistake. Many Windows users can not install
-the package if there is at least one non-ASCII character (e.g. emoji)
-in the ``README.md`` file which is encoded in UTF-8.
+For example, using ``long_description = open("README.md").read()`` in
+``setup.py`` is a common mistake. Many Windows users cannot install
+such packages if there is at least one non-ASCII character (e.g. emoji)
+in their UTF-8-encoded ``README.md`` file.
 
-For example, 489 packages of the 4000 most downloaded packages from
-PyPI used non-ASCII characters in README. And 82 packages of them
-can not be installed from source package when locale encoding is
-ASCII. [1]_ They used the default encoding to read README or TOML
-file.
+Of the 4000 most downloaded packages from PyPI, 489 use non-ASCII
+characters in their README, and 82 of these do not explicitly specify
+the encoding used to open it. [1]_ Therefore, they cannot be installed
+from source in non-UTF-8 locales.
 
 Another example is ``logging.basicConfig(filename="log.txt")``.
-Some users expect UTF-8 is used by default, but locale encoding is
-used actually. [2]_
+Some users might expect it to use UTF-8 by default, but the locale
+encoding is actually what is used. [2]_
 
-Even Python experts assume that default encoding is UTF-8.
-It creates bugs that happen only on Windows. See [3]_, [4]_, [5]_,
+Even Python experts may assume that the default encoding is UTF-8.
+This creates bugs that only happen on Windows; see [3]_, [4]_, [5]_,
 and [6]_ for example.
 
-Emitting a warning when the ``encoding`` option is omitted will help
-to find such mistakes.
+Emitting a warning when the ``encoding`` argument is omitted will help
+find such mistakes.
 
 
 Explicit way to use locale-specific encoding
 --------------------------------------------
 
 ``open(filename)`` isn't explicit about which encoding is expected:
 
-* Expects ASCII (not a bug, but inefficient on Windows)
-* Expects UTF-8 (bug or platform-specific script)
-* Expects the locale encoding.
+* If ASCII is assumed, this isn't a bug, but is inefficient on Windows
```

Comment:
Could you clarify what you mean by "inefficient on Windows"? Per the [docs](https://docs.python.org/3/library/codecs.html#standard-encodings), both Latin-1 (the default for a plurality/majority of Windows machines) and `mbcs` (defined as the locale default) take the fast path. Is there a significant speed penalty Latin-1 vs. ASCII, or what is this referring to?

Reply:
On my Windows environment, cp932 is used and it is much slower than UTF-8.
Maybe, on Windows environments using latin-1 may be not cause slowdown.

Reply:
```suggestion
* If ASCII is assumed, this isn't a bug, but may result in decreased
  performance on Windows, particularly with non-Latin-1 locale encodings
```

Gotcha. What about this?

Reply:
LGTM

Reply:
:+1: Feel free to apply it